### ISSUE TITLE
Will the AREA Research Agenda have "pure" use cases as research topics?

### ISSUE THREAD

**Issue by theareaorg** [theareaorg/AREA-Research-Agenda#27]
On March 1, a use case was suggested as a research topic. Specifically, it is the [visualization of data from sensors on a connected tool](https://github.com/theareaorg/AREA-Research-Agenda/blob/main/AREA_Research_Agenda_2021/Categories_and_Topics/Research_Topics/Bintegration5-connectedtools.adoc). 

The use case was captured because the source was a valued AREA member, however, in my mind this is a specific example of a bigger topic of visualization of data from any sensors. There's another example of the same problem in a topic written about Oil and Gas industry in which the user is able to visualize data from chemical and/or radiation sensors. The research topic is [here](https://github.com/theareaorg/AREA-Research-Agenda/blob/main/AREA_Research_Agenda_2021/Categories_and_Topics/Research_Topics/Ioil5-sensors.adoc).

This will be difficult to avoid entirely, so maybe we actually should go the other way and embrace it? 

Would love to hear others think and proposed resolution for management of use cases and this issue?

**Comment by bookwolf** [comment]
I tend to agree. You would find the same use case in metals and mining, and I'm sure other parallel industries, as well. I think this once again makes a case for multi-tagging.
